apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -139,6 +139,41 @@ export interface IConfigApiReport {
    * @defaultValue `false`
    */
   includeForgottenExports?: boolean;
+
+  /**
+   * Specifies a list of {@link https://tsdoc.org/ | TSDoc} tags that should be reported in the API report file for
+   * items whose documentation contains them.
+   *
+   * @remarks
+   * Tag names must begin with `@`.
+   *
+   * This list may include standard TSDoc tags as well as custom ones.
+   * For more information on defining custom TSDoc tags, see
+   * {@link https://api-extractor.com/pages/configs/tsdoc_json/#defining-your-own-tsdoc-tags | here}.
+   *
+   * Note that an item's release tag will always reported; this behavior cannot be overridden.
```

Comment:
@Josmithr If you try to specify a release tag in the `tagsToReport` structure, we should report an error then.  Currently it seems to be ignored.

Reply:
Good call. Will update.